### PR TITLE
FEA-2972: Dont publish internally

### PR DESCRIPTION
**Pull request by matthewnitschke-wk**
# [FEA-2972](https://jira.atl.workiva.net/browse/FEA-2972)
Issue Status

## Motivation

This is an OSS repo, that we use publicly: https://pub.dev/packages/sockjs_client_wrapper, but the dockerfile still is setup to publish internally

## Changes
- cleans up the dockerfile to only run pub-get for dependency scanning
- also cleans up some gha things

### QA Checklist
- [ ] CI passes should be sufficient

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Review by robbecker-wf — APPROVED** [review]

**Comment by robbecker-wf** [comment]
QA+1 CI passes

File: skynet.yaml
Comment (evanweible-wf) [inline skynet.yaml:1]: We may want to keep this file for now, but with an empty config that just passes. Right now, core checks ensure that we get a skynet run created, but that could change

**Review by evanweible-wf — DISMISSED** [review]

File: skynet.yaml
Comment (matthewnitschke-wk) [reply skynet.yaml:1]: Ah, shoot, good point, I'll re-add

**Review by evanweible-wf — APPROVED** [review]

**Review by robbecker-wf — APPROVED** [review]
QA+1 CI passes again .. now with more skynet

**Comment by matthewnitschke-wk** [comment]
🚀 @Workiva/release-management-p 🚢

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**